apps/web-giddh/src/app/gst/gst.component.scss

```diff
@@ -405,15 +405,24 @@ hr {
 .header-info .btn-group > .dropdown-menu {
     margin-top: 0;
 }
+::ng-deep #monthPicker button.btn.btn-default.btn-info {
+    background: $chetwode-blue !important;
+    color: #fff!important;
```

Comment:
Add color variable

Reply:
to be covered with code designing @shalinee-walkover

Reply:
ok